docs/guides/mysql/gitops/gitops.md

```
@@ -378,7 +378,7 @@ mysql> show variables like 'read_buffer_size';
 
 ```
 At first, we will create a secret containing `user.conf` file with required configuration settings.
-To know more about this configuration file, check [here](/docs/guides/mysql/configuration/using-config-file.md)
+To know more about this configuration file, check [here](/docs/guides/mysql/configuration/config-file/index.md)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)